src/assembler/parser/index.ts

```diff
@@ -12,6 +12,8 @@ const INSTRUCTION = `([a-z]+)${SPACE_OR_TAB}+(${OPTION}(?:${SPACE_OR_TAB}*,${SPA
 const LITERAL_SYMBOL_DECLARATION = `(${SYMBOL})${SPACE_OR_TAB}+EQU${SPACE_OR_TAB}+(${VALUE})`
 const AREA_DECLARATION = `AREA${SPACE_OR_TAB}+(${SYMBOL})${SPACE_OR_TAB}*,${SPACE_OR_TAB}*(DATA|CODE)${SPACE_OR_TAB}*,${SPACE_OR_TAB}*(READ(WRITE|ONLY))`
 const COMMENT = `;[^\\n]*`
+const SPACE_OR_FILL = `SPACE|FILL|\\%`
+const SPACE_OR_FILL_Instruction = `${SPACE_OR_TAB}*(${SPACE_OR_FILL})${SPACE_OR_TAB}+([0-9]+\\*[0-9]+)`
```

Comment:
make all upper case since it is a constant

Reply:
Changed it